### PR TITLE
Us057 change respondent name and number

### PR DESCRIPTION
**Pull request by andrewmil**
Added tests for US057 Change respondent name and number
Works with current master branches

[Trello](https://trello.com/c/1ZGcBo4W/45-us057-int-change-respondent-name-number-m)
[Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/248512717/US057+Change+Respondent+Name+and+Number+M)

### CODE REVIEW

File: acceptance_tests/features/pages/edit_respondent_details_form.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+def edit_first_name():
+    browser.find_by_id('firstName').fill('Jacky')
+
+
+def edit_last_name():
+    browser.find_by_id('firstName').fill('Turner')
+
+
+def fill_first_name(characters):
```
Comment (emilioward) [inline acceptance_tests/features/pages/edit_respondent_details_form.py]: Is this used anywhere?

File: acceptance_tests/features/pages/edit_respondent_details_form.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    browser.find_by_id('firstName').fill("x" * characters)
```
Comment (emilioward) [inline acceptance_tests/features/pages/edit_respondent_details_form.py]: Is this used anywhere?

Comment (andrewmil) [reply acceptance_tests/features/pages/edit_respondent_details_form.py]: Nope, I'll remove

File: acceptance_tests/features/pages/edit_respondent_details_form.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
+from acceptance_tests import browser
+
+
+def edit_first_name():
+    browser.find_by_id('firstName').fill('Jacky')
+
+
+def edit_last_name():
+    browser.find_by_id('firstName').fill('Turner')
```
Comment (tej99) [inline acceptance_tests/features/pages/edit_respondent_details_form.py]: browser.find_by_id(**'lastName'**)?

File: acceptance_tests/features/steps/edit_respondent_details.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    edit_respondent_details_form.click_save()
+
+
+@when('they decide to cancel')
+def cancel_changes(_):
+    edit_respondent_details_form.click_cancel()
+
+
+@then('they are navigated back to the RU Details page')
+@then('they are able to enter up to 254 characters')
```
Comment (tej99) [inline acceptance_tests/features/steps/edit_respondent_details.py]: Does this then statement make sense?

**Comment by tej99** [comment]
the asserts in view_reporting_unit_details.py lines 78-82 will need changing according what the previous tests change them to as the database is not reset after each feature.

File: acceptance_tests/features/steps/edit_respondent_details.py (hunk 4 above)
Comment (emilioward) [reply acceptance_tests/features/steps/edit_respondent_details.py]: I saw that too and couldn't make sense of it either

File: resources/sample_files/business-survey-sample-date.csv [hunk 5]
```diff
@@ -1,4 +1,4 @@
-49900000001:F:50300:50300:45320:45320:8478:801325:9900000576:1:E:FE:01/09/1993:ENTNAME1_COMPANY1:ENTNAME2_COMPANY1::RUNAME1_COMPANY1:RUNNAME2_COMPANY1::TOTAL UK ACTIVITY:::C:D:7:0001:S
```
Comment (tej99) [inline resources/sample_files/business-survey-sample-date.csv]: I think you pushed this by mistake

File: acceptance_tests/features/steps/edit_respondent_details.py (hunk 4 above)
Comment (jcox-dev) [reply acceptance_tests/features/steps/edit_respondent_details.py]: Something to flag with RAD as it's copied from Confluence.

**Comment by andrewmil** [comment]
[US057: INT Change Respondent Name & Number [M]](https://trello.com/c/1ZGcBo4W/45-us057-int-change-respondent-name-number-m)

**Comment by andrewmil** [comment]
[US057: INT Change Respondent Name & Number [M]](https://trello.com/c/1ZGcBo4W/45-us057-int-change-respondent-name-number-m)

**Pull request merged**